tests/Unit/Controller/Admin/PageExportControllerTest.php

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\ImportExportBundle\Tests\Unit\Controller\Admin;
+
+use Neusta\Pimcore\ImportExportBundle\Controller\Admin\PageExportController;
+use Neusta\Pimcore\ImportExportBundle\Documents\Export\PageExporter;
+use Neusta\Pimcore\ImportExportBundle\Toolbox\Repository\PageRepository;
+use PHPUnit\Framework\TestCase;
+use Pimcore\Model\Document\Page;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Prophecy\Prophecy\ObjectProphecy;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class PageExportControllerTest extends TestCase
+{
+    use ProphecyTrait;
+
+    private PageExportController $controller;
+
+    /** @var ObjectProphecy<PageExporter> */
+    private $pageExporter;
+
+    /** @var ObjectProphecy<PageRepository> */
+    private $pageRepository;
+    private Request $request;
+
+    protected function setUp(): void
+    {
+        $this->pageExporter = $this->prophesize(PageExporter::class);
+        $this->pageRepository = $this->prophesize(PageRepository::class);
+
+        $this->controller = new PageExportController(
+            $this->pageExporter->reveal(),
+            $this->pageRepository->reveal(),
+        );
+        $this->request = new Request(['page_id' => 17]);
+    }
+
+    public function testExportPage_regular_case(): void
+    {
+        $page = $this->prophesize(Page::class);
+        $this->pageRepository->getById(17)->willReturn($page->reveal());
+        $this->pageExporter->toYaml($page->reveal())->willReturn('TEST_YAML');
+
+        $response = $this->controller->exportPage($this->request);
+
+        self::assertTrue($response instanceof Response);
```

Comment:
```suggestion
        self::assertInstanceOf(Response::class, $response);
```

However, this is unnecessary as the method has a return type of `Response`, so this will *always* be the case.